schema/data/fixtures/reporter-all-access-teardown.sql

```diff
@@ -1,5 +1,8 @@
 begin;
 
+-- Re-populate products table, which was truncated in test setup:
+\i ../schema/data/dev/product.sql
+
```

Comment:
This path is relative to the folder from which the `cypress run` command is run, which is the same in all environments thanks to npm/yarn scripts. I recognize it's kind of ugly, because it doesn't make a lot of sense when viewed alone as part of this fixture.

We can either leave the path hard-coded like this (which seems a bit gross, but has the advantage of being very straightforward), or we can use a variable like so:
```
\set product_seeds :DATA_DIR '/dev/product.sql'
\i :product_seeds
```
...by adding a variables flag to the psql script in `cypress/support/commands.js`:
```
psql -v DATA_DIR=$(pwd)/../schema/data -d ciip_portal_dev
```


What do you think @matthieu-foucault?

Reply:
I don't think that the teardown should be responsible for re-populating the products table, which other tests may or may not need. We can leave it as-is for now, but as @dleard mentioned offline, there is a need for rewriting our e2e fixtures. We're currently assuming that the `dev` data is deployed when starting a test, but I think that each e2e test file should assume an empty database, and load the fixtures it needs in a `before`/`beforeEach` hook

Reply:
I will be adding a card in our issue tracker for that fixture refactor

Reply:
I suppose I was operating on the principle that tests should be independent of each other - teardown should undo what's done in setup. There are other test suites that use this setup/teardown, so the idea was to fix the side effect each place it's used.

But certainly agreed that they all need to be re-conceived and re-organized.

Reply:
Yes, tests should not have side effect, but I would add that they should not assume that the database contains any data. We currently have a fair amount of data added before we start cypress via https://github.com/bcgov/cas-ciip-portal/blob/3e42f82e4fafbf83c130290f274ade7051727fa8/.circleci/config.yml#L380
which is not a great practice and will be changed when we tackle https://youtrack.button.is/issue/GGIRCS-1908